### DISCUSSION TITLE
Helix-editor integration?

### DISCUSSION THREAD

**Discussion by cywil5115** [nvim-neorg/neorg#1452] [Ideas]
It would be cool to have neorg in helix as an option for note-taking. I know that helix has basic .org implementation but norg would be so nice to have.

**Comment by katawful** [comment]
basic norg file format support can be easily integrated with the tree-sitter parser

neorg is a neovim specific plugin for norg, so helix support would have to be made from scratch

the same holds true for org-mode and its filetype. many features are exclusive to emacs
